src/main/java/uk/gov/companieshouse/service/documentRender/DocumentRenderClient.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.companieshouse.service.documentRender;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import uk.gov.companieshouse.config.DocumentRenderConfig;
+import uk.gov.companieshouse.model.dto.documentRender.DissolutionCertificateData;
+
+@Service
+public class DocumentRenderClient {
+
+    private static final String DOCUMENT_RENDER_STORE_ENDPOINT = "/document-render/store";
+
+    private static final String HEADER_AUTHORIZATION = "Authorization";
+    private static final String HEADER_ASSET_ID = "assetID";
+    private static final String HEADER_TEMPLATE_NAME = "templateName";
+    private static final String HEADER_ACCEPT = "Accept";
+    private static final String HEADER_CONTENT_TYPE = "Content-Type";
+    private static final String HEADER_LOCATION = "Location";
+
+    private static final String HEADER_ASSET_ID_VALUE = "dissolution";
+    private static final String HEADER_ACCEPT_VALUE = "application/pdf";
+    private static final String HEADER_CONTENT_TYPE_VALUE = "text/html";
+
+    private final DocumentRenderConfig config;
+
+    public DocumentRenderClient(DocumentRenderConfig config) {
+        this.config = config;
+    }
+
+    public String generateAndStoreDocument(DissolutionCertificateData data, String templateName, String location) {
+        return WebClient
+                .create(config.getDocumentRenderHost())
+                .post()
+                .uri(DOCUMENT_RENDER_STORE_ENDPOINT)
+                .header(HEADER_AUTHORIZATION, config.getApiKey())
+                .header(HEADER_ASSET_ID, HEADER_ASSET_ID_VALUE)
+                .header(HEADER_TEMPLATE_NAME, templateName)
+                .header(HEADER_ACCEPT, HEADER_ACCEPT_VALUE)
+                .header(HEADER_CONTENT_TYPE, HEADER_CONTENT_TYPE_VALUE)
+                .header(HEADER_LOCATION, location)
+                .body(Mono.just(asJsonString(data)), String.class)
```

Comment:
I had to serialise the DissolutionCertificateData into a String as Spring was complaining that I was providing a Content-Type of `text/html` but trying to send JSON.

Although Spring is correct, the Document Render Service requires a Content-Type of `text/html` to be submitted in order for HTML -> PDF conversion to take place - see [here](https://github.com/companieshouse/document-render-service/blob/master/helpers/convert.go#L35)